.github/workflows/continuous-integration.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: [10, 12, 14, 16]
+        node-version: [12, 14, 16]
```

Comment:
Could we do this in a separate PR? If the CI pipeline is only failing with v10, I think it should be safe to merge.

This would also be considered a breaking change. Since this package doesn't get much feature updates, it may be preferable to follow Module LTS Policy (i.e. to drop support for v10 when v16 reaches EOL).

For NPM compatibility, we may want to explicitly set the NPM version (i.e `npm install --global npm@7` for our CI pipelines.

I'll prepare PRs to fix this issue across all of our existing CI pipelines.

Reply:
I've opened https://github.com/loopbackio/cicd/issues/6 to track this NPM version pinning issue.

Reply:
sure. I've updated this PR only have the codeowners file change.